### PR TITLE
l10n: zh_CN: for git 2.44 rnd1

### PR DESCRIPTION
**Pull request by dyrone**
Thanks for taking the time to contribute to Git! Please be advised that the
Git community does not use github.com for their contributions. Instead, we use
a mailing list (git@vger.kernel.org) for code submissions, code reviews, and
bug reports. Nevertheless, you can use GitGitGadget (https://gitgitgadget.github.io/)
to conveniently send your Pull Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

**Comment by dyrone** [comment]
replay子命令中引入了新的advance选项， 我目前翻译为“演进”

**Comment by jiangxin** [comment]
Hi, A new translatable message has been introduced in Git upstream, please update your translations.

**Comment by jiangxin** [comment]
Commit ["diff: mark param1 and param2 as placeholders"](https://lore.kernel.org/git/3a82f72f33663f162aa41cb20c0fb3b6786971c9.1707900029.git.worldhello.net@gmail.com/) has been merged, and two translatable messages have been changed. Let's start the 3rd (and perhaps final) round of l10n.

**Comment by dyrone** [comment]
> Commit ["diff: mark param1 and param2 as placeholders"](https://lore.kernel.org/git/3a82f72f33663f162aa41cb20c0fb3b6786971c9.1707900029.git.worldhello.net@gmail.com/) has been merged, and two translatable messages have been changed. Let's start the 3rd (and perhaps final) round of l10n.

Sure, a updated patch has been posted.

**Comment by dyrone** [comment]
> replay子命令中引入了新的advance选项， 我目前翻译为“演进”

台湾的本地化协调者，同样翻译为"演進"

**Comment by jiangxin** [comment]
Your commit log is not a complete message.

    l10n: zh_CN: for git 2.44 rnd

rnd3?

File: po/zh_CN.po [hunk 1]
```diff
@@ -13956,6 +13989,86 @@ msgstr "--convert-graft-file 不带参数"
...
+msgid "need some commits to replay"
+msgstr "需要一些提交来重放"
+
+#: builtin/replay.c
+msgid "--onto and --advance are incompatible"
+msgstr "--onto 和 --advance 不兼容"
+
+#: builtin/replay.c
+msgid "all positive revisions given must be references"
+msgstr "提供的所有阳性版本必须为引用"
```
Comment (jiangxin) [inline po/zh_CN.po:14002]: positive revisions 译为 `正向版本` 。而 `负向版本` 指的是 `^revison`。

File: po/zh_CN.po [hunk 2]
```diff
@@ -13956,6 +13989,86 @@ msgstr "--convert-graft-file 不带参数"
...
+#: builtin/replay.c
+msgid "cannot implicitly determine correct base for --onto"
+msgstr "不能隐式地确定 --onto 正确的基线"
+
+#: builtin/replay.c
+msgid ""
+"(EXPERIMENTAL!) git replay ([--contained] --onto <newbase> | --advance "
+"<branch>) <revision-range>..."
+msgstr ""
+"（还处于试验中的特性！）git replay ([--contained] --onto <新基线> | --"
```
Comment (jiangxin) [inline po/zh_CN.po:14034]: `（还处于试验中的特性！）` 改成 `（试验中!）`。
1. 使用半角感叹号更紧凑。
2. 像原文一样简洁，用户可理解。

File: po/zh_CN.po [hunk 3]
```diff
@@ -13956,6 +13989,86 @@ msgstr "--convert-graft-file 不带参数"
...
+"will be forced"
+msgstr "'%s' 位在'struct rev_info'中的一些版本遍历的选项将被强制覆盖"
+
+#: builtin/replay.c
+msgid "error preparing revisions"
+msgstr "准备版本时错误"
+
+#: builtin/replay.c
+msgid "replaying down to root commit is not supported yet!"
+msgstr "目前还不支持重放到根提交!"
```
Comment (jiangxin) [inline po/zh_CN.po:14066]: 使用全角感叹号，和下面的翻译保持一致

File: po/zh_CN.po [hunk 4]
```diff
@@ -13956,6 +13989,86 @@ msgstr "--convert-graft-file 不带参数"
...
+#: builtin/replay.c
+msgid "option --onto or --advance is mandatory"
+msgstr "选项 --onto 或 --advance 必须指定其一"
+
+#: builtin/replay.c
+#, c-format
+msgid ""
+"some rev walking options will be overridden as '%s' bit in 'struct rev_info' "
+"will be forced"
+msgstr "'%s' 位在'struct rev_info'中的一些版本遍历的选项将被强制覆盖"
```
Comment (jiangxin) [inline po/zh_CN.po:14058]: 一些版本遍历选项将被覆盖，如 'struct rev_info' 中的 '%s' 位将被强制设定

File: po/zh_CN.po [hunk 5]
```diff
@@ -17488,6 +17580,10 @@ msgstr "打包仓库中未打包对象"
 msgid "Create, list, delete refs to replace objects"
 msgstr "创建、列出、删除对象替换引用"
 
+#: command-list.h
+msgid "EXPERIMENTAL: Replay commits on a new base, works with bare repos too"
+msgstr "处于试验中的特性：基于一个新基线重放提交，同样适用于纯仓库"
```
Comment (jiangxin) [inline po/zh_CN.po:17585]: 试验中：...

File: po/zh_CN.po [hunk 6]
```diff
@@ -17805,6 +17901,37 @@ msgstr "一个管理大型 Git 仓库的工具"
...
+msgid "commit-graph fanout values out of order"
+msgstr "提交图形的扇出值失序"
+
+#: commit-graph.c
+msgid "commit-graph OID lookup chunk is the wrong size"
+msgstr "提交图形的对象 ID 查询块大小错误"
+
+#: commit-graph.c
+msgid "commit-graph commit data chunk is wrong size"
+msgstr "提交图形的数据块大小错误"
```
Comment (jiangxin) [inline po/zh_CN.po:17918]: 提交图形的提交数据块大小错误

File: po/zh_CN.po [hunk 7]
```diff
@@ -17825,6 +17952,18 @@ msgstr "提交图形哈希版本 %X 和版本 %X 不匹配"
...
+msgid "commit-graph required OID fanout chunk missing or corrupted"
+msgstr "提交图形所需的对象 ID 扇出块缺失或损坏"
+
+#: commit-graph.c
+msgid "commit-graph required OID lookup chunk missing or corrupted"
+msgstr "提交图形所需的对象 ID 查询块缺失或损坏"
+
+#: commit-graph.c
+msgid "commit-graph required commit data chunk missing or corrupted"
+msgstr "提交图形所需的提交块缺失或损坏"
```
Comment (jiangxin) [inline po/zh_CN.po:17965]: s/提交块/提交数据块/

File: po/zh_CN.po [hunk 8]
```diff
@@ -17805,6 +17901,37 @@ msgstr "一个管理大型 Git 仓库的工具"
 msgid "commit-graph file is too small"
 msgstr "提交图形文件太小"
 
+#: commit-graph.c
+msgid "commit-graph oid fanout chunk is wrong size"
+msgstr "提交图形中对象 ID 的扇出块大小错误"
```
Comment (jiangxin) [inline po/zh_CN.po:17906]: commit-graph 是翻译为提交图，还是提交图形？我倾向于前者。这个要在 HEAD 术语中明确一下。比如 photo 和 photograph 都翻译为照片，没有图形两个字。

File: po/zh_CN.po [hunk 9]
```diff
@@ -21469,6 +21612,15 @@ msgstr "多包索引包名无序：'%s' 在 '%s' 之前"
 msgid "bad pack-int-id: %u (%u total packs)"
 msgstr "错的 pack-int-id：%u（共有 %u 个包）"
 
+#: midx.c
+msgid "MIDX does not contain the BTMP chunk"
+msgstr "多包索引中未包含位图索引块"
```
Comment (jiangxin) [inline po/zh_CN.po:21617]: 是不是 BTMP 不翻译更好？

File: po/zh_CN.po [hunk 10]
```diff
@@ -27329,6 +27469,11 @@ msgstr "另外，您的索引中包含未提交的变更。"
 msgid "cannot %s: Your index contains uncommitted changes."
 msgstr "不能%s：您的索引中包含未提交的变更。"
 
+#: xdiff-interface.c
+#, c-format
+msgid "unknown style '%s' given for '%s'"
+msgstr "键 '%2$s' 的未知冲突输出风格取值 '%1$s'"
```
Comment (jiangxin) [inline po/zh_CN.po:27475]: 为了避免该字符串重用导致翻译错误，原位未体现的“冲突输出" 不比翻译出来，虽然看了原文知道是这么回事。

'%2$s' 的未知风格取值 '%1$s'

**Review by jiangxin — COMMENTED** [review]
发现一些翻译的问题，待商榷

File: po/zh_CN.po (hunk 8 above)
Comment (dyrone) [reply po/zh_CN.po:17906]: 我倾向前者，但是不是因为graph是否必须翻译为图形的问题，之前commit-graph翻译的就叫做“提交图形”。

photo和photograph的主要区别是，后者可以作为verb

**Comment by dyrone** [comment]
> Your commit log is not a complete message.
> 
> ```
> l10n: zh_CN: for git 2.44 rnd
> ```
> 
> rnd3?

是的，应该加个s

File: po/zh_CN.po (hunk 9 above)
Comment (dyrone) [reply po/zh_CN.po:21617]: 貌似是的， 我改下

File: po/zh_CN.po [hunk 11]
```diff
@@ -1859,6 +1861,11 @@ msgstr "选项 '%s' 需要 '%s'"
 msgid "Unexpected option --output"
 msgstr "未知参数 --output"
 
+#: archive.c
+#, c-format
+msgid "extra command line parameter '%s'"
+msgstr "添加了额外的命令行参数：'%s'"
```
[the remark this answers is not in the document]
Comment (dyrone) [reply po/zh_CN.po]: 已修复

File: po/zh_CN.po [hunk 12]
```diff
@@ -9990,6 +10013,13 @@ msgstr ""
 "git merge-file [<选项>] [-L <名字1> [-L <初始名字> [-L <名字2>]]] <文件1> <初"
 "始文件> <文件2>"
 
+#: builtin/merge-file.c diff.c
+msgid ""
+"option diff-algorithm accepts \"myers\", \"minimal\", \"patience\" and "
+"\"histogram\""
+msgstr ""
+"diff-algorithm 选项有 \"myers\"、\"minimal\"、\"patience\" 和 \"histogram\""
```
[the remark this answers is not in the document]
Comment (dyrone) [reply po/zh_CN.po]: 已修复

File: po/zh_CN.po (hunk 2 above)
Comment (dyrone) [reply po/zh_CN.po]: 已修复

File: po/zh_CN.po (hunk 10 above)
Comment (dyrone) [reply po/zh_CN.po]: 已修复

File: po/zh_CN.po (hunk 8 above)
Comment (dyrone) [reply po/zh_CN.po:17903]: 这次先保持原样吧，如果“提交图”ok，单独做个提交改一下

**Pull request merged**